### PR TITLE
Updated Catalan translation

### PR DESCRIPTION
**Pull request by jcatfor**
Catalan translation with many corrections and translations that make more sense

### CODE REVIEW

**Comment by vixalien** [comment]
cc @gforcada need your review

**Comment by vixalien** [comment]
you will also need to add your name at `msgid "translator-credits"`

**Comment by gforcada** [comment]
The translation in general looks good 👍🏾 though there are a few stylistic changes that I would want to double check, unfortunately I'm rather busy right now, hopefully within this week I have time to do a proper review

**Comment by jcatfor** [comment]
> The translation in general looks good 👍🏾 though there are a few stylistic changes that I would want to double check, unfortunately I'm rather busy right now, hopefully within this week I have time to do a proper review



> you will also need to add your name at `msgid "translator-credits"`

Ok, thank you. I'll wait until @gforcada does the review to add my name

File: po/ca.po [hunk 1]
```diff
@@ -2,61 +2,64 @@
...
 msgid "Pin notes to your desktop"
-msgstr ""
+msgstr "Fixeu notes al vostre escriptori"
 
 #: data/com.vixalien.sticky.desktop.in.in:4
 #: data/com.vixalien.sticky.appdata.xml.in.in:6 data/ui/notes.ui:32
 #: src/application.ts:68 src/application.ts:337
 msgid "Sticky Notes"
-msgstr "Notes enganxoses"
+msgstr "Notes adhesives"
```
Comment (gforcada) [inline po/ca.po:32]: Al [recull terminològic de Softcatalà](https://static.softcatala.org/terminology/sc-glossary.html) es recommana `fixat` per `sticky` amb aquestes altres opcions:

- fixat (usada 57.14%, coincidències 4)
- enganxós (usada 28.57%, coincidències 2)
- enganxós | enganxifós | apegalós | pegallós (usada 14.29%, coincidències 1)

Em sembla bona idea `adhesiu` tot i així 👍🏾

File: po/ca.po [hunk 2]
```diff
@@ -2,61 +2,64 @@
...
+msgstr "Adhesives;Notes;Escriure;"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:9
 msgid ""
 "Sticky Notes is a simple sticky notes app for GNOME. It allows you to create "
 "notes and format them."
 msgstr ""
-"El Notes enganxoes és una aplicació senzilla per prendre notes pel GNOME.Us "
-"permet crear notes i formatar-les."
+"Notes Adhesives és una senzilla aplicació de notes adhesives pel GNOME. Et "
```
Comment (gforcada) [inline po/ca.po:44]: De nou, normalment a Softcatalà, es fa servir `Us` i no `Et` per referir-se a l'usuari, si prefereixes deixar el canvi, s'hauria de revisar tota la traducció llavors

File: po/ca.po [hunk 3]
```diff
@@ -2,61 +2,64 @@
...
 #: data/com.vixalien.sticky.appdata.xml.in.in:16
 msgid "notes are restored if they were open when the application was closed"
 msgstr ""
-"es tornen a obrir les notes si estavan obertes quan es va tancar l'aplicació"
+"les notes es restauren si estaven obertes quan es va tancar l'aplicació"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:17
 msgid "changing color of notes"
-msgstr "canviar el color de les notes"
+msgstr "canvi de color de les notes"
```
Comment (gforcada) [inline po/ca.po:62]: Crec que aquí la meva versió és més correcta, ja que aquesta cadena de text només apareix a la configuració interna del programa, segurament és un valor binari, una opció que et permet decidir si vols que es canviïn els colors de la nota.

File: po/ca.po [hunk 4]
```diff
@@ -469,31 +479,30 @@ msgstr "Carbó"
...
 #: src/util.ts:297
-#, fuzzy
 msgid "Delete"
-msgstr "Suprimeix la nota"
+msgstr "Suprimir"
 
 #: src/view.ts:351
 msgid "Empty Note"
-msgstr "Nota buida"
+msgstr "Nota Buida"
```
Comment (gforcada) [inline po/ca.po:505]: Normalment en català la capitalització de totes les paraules no s'utilitza massa, almenys a l'entorn d'escriptori GNOME

File: po/ca.po [hunk 5]
```diff
@@ -469,31 +479,30 @@ msgstr "Carbó"
 
 #: src/util.ts:81
 msgid "Window"
-msgstr "Finestra"
+msgstr "de la Finestra"
```
Comment (gforcada) [inline po/ca.po:482]: 🤔 seria `Finestra` a seques, no?

**Review by gforcada — COMMENTED** [review]
The direction is good, a few stylistic choices could be change (to my taste), so if @jcatfor discards all my comments, the translation is still good to get merged 👍🏾

**Comment by vixalien** [comment]
@jcatfor please review the requested changes

**Comment by vixalien** [comment]
@jcatfor  could you care to take a look at this PR again and rebase?

**Comment by vixalien** [comment]
@jcatfor

**Pull request closed without merging**